app/models/rental.rb

```diff
@@ -86,6 +86,36 @@ def times
   end
   alias dates times
 
+  def event_name
+    "#{item_type.name}(#{item_type.id}) - Rental ID: #{id}"
+  end
+
+  def event_status_color
+    case rental_status
+    when 'reserved'
+      return '#0092f'
+    when 'checked_out'
+      return '#f7ff7'
+    when 'checked_in'
+      return '#09ff00'
+    else
+      return '#000000' # black signifies a non event status
+    end
```

Comment:
Yes, it certainly should be within the CSS side. However, since I'm working with the gon gem, it will be hacky to break down the gon variable within the rental.js to insert the color. The color is passed in as json, e.g. events: [{title: "1", date: "1-2-16", color: "fff130" }], so no CSS is directly involved. We can discuss if there is a better option?